test/Common.cpp

```diff
@@ -239,4 +229,34 @@ bool isValidSemanticTestPath(boost::filesystem::path const& _testPath)
 	return true;
 }
 
+bool loadVMs(CommonOptions const& _options)
+{
+	try
+	{
+		auto [evmSupport, ewasmSupport] = solidity::test::EVMHost::checkVmPaths(_options.vmPaths);
```

Comment:
It would be easier to understand what these two mean if we had a type here. I'd at least slightly rename this to make it more apparent they are booleans:

```suggestion
		auto [evmSupported, ewasmSupported] = solidity::test::EVMHost::checkVmPaths(_options.vmPaths);
```

Reply:
You can't type structured bindings :-). But I can rename them.

Reply:
I know. That's why I suggested the name change. I'm just saying that for me it is a case where seeing the types would be helpful.